### ISSUE TITLE
Colour picker source limited to primary monitor

### ISSUE THREAD

**Issue by ZaLiTHkA** [motorlatitude/Drop#81]
howdy, I only discovered this one the other day, loving it so far! I do have one minor annoyance though: I run a dual monitor setup and I cannot "pick" any colour from my secondary monitor.

so far, I've only tested this under Windows 10 (version 21H2, OS build 19044.1466). I do run Pop!_OS 21.10 on my personal laptop, so I'll see if I can attach a second monitor to test there as well.

---

_edit:_ I just switched my monitors around, so I now have them as `[1][2]` (with "primary" on the left), instead of `[2][1]`. now Drop allows me to pick any pixel across either screen. which prompted me to test with Windows configured to have monitors stacked vertically instead..

basically, this bug only seems to affect multi-monitor setups where to the secondary is either _above_  or to the _left_ of the primary. when the secondary is _below_ or to the _right_ everything works perfectly.

---

I didn't see any other issues logged for this, so just wanted to check: is this a known issue or should I gather some more information for you?

guess I should also ask: is this project still in development? looks like things have been pretty quiet here for the last 2-odd years... 🤔
